### PR TITLE
Windows support

### LINKED ISSUE #10: Support `install()` for Windows
The `install()` function is implemented only for Linux now. It would be nice to provide the same functionality for other OS as well.

https://github.com/petvana/AutoSysimages.jl/blob/a28a424b9dae71d0ca6ae14515da73749185fb14/src/AutoSysimages.jl#L388-L389

### PR DESCRIPTION
**Pull request by curio-sitas**
I added the windows julia bin directory (`Sys.BINDIR`) (wich should work on all os) and also added the windows support in the install function.

Finally, i formated the main file using the JuliaFormatter package.

All works as intended on my machine (Win11), including tests and asysimg command available in the terminal

Changes proposal : 
- use `Sys.BINDIR` for all OS

### CODE REVIEW

File: src/AutoSysimages.jl [hunk 1]
```diff
@@ -156,7 +160,8 @@ function build_sysimage(background::Bool = false)
         if isinteractive()
             @info """AutoSysimages: No project settings (SysimagePreferences.toml) found.
 Do you want to select packages to be included now?"""
-            if request(RadioMenu(["Yes (select packages)", "No (include all packages)"])) == 1
+            if request(RadioMenu(["Yes (select packages)", "No (include all packages)"])) ==
+               1
```
Comment (petvana) [inline src/AutoSysimages.jl]: Thanks for formatting. However, here, i wouldn't split the line not to have a single symbol on the next line.

File: src/AutoSysimages.jl [hunk 2]
```diff
@@ -389,36 +393,44 @@ end
...
+
     if Sys.islinux() || Sys.isapple()
-        dir_exists = ispath(dir)
-        dir_exists || mkpath(dir)
         script = joinpath(@__DIR__, "..", "scripts", "linux", "asysimg")
         script = abspath(normpath(script))
-        cp(script, joinpath(dir, "asysimg"), force=true)
+        cp(script, joinpath(dir, "asysimg"), force = true)
+    elseif Sys.iswindows()
+        script = joinpath(@__DIR__, "..", "scripts", "windows", "asysimg.bat")
```
Comment (petvana) [inline src/AutoSysimages.jl:404]: This is kind of duplicite code for Linux and Windows (except for the extension). We can merge together.

File: src/AutoSysimages.jl (hunk 1 above)
Comment (curio-sitas) [reply src/AutoSysimages.jl]: Ok, i will no more format on this repo then ;)

**Comment by petvana** [comment]
Thank you for your contribution. The `Sys.BINDIR` is what I was looking for. Nevertheless, this directory would not work for Linux and MacOS because setting `PATH` is optional.

https://julialang.org/downloads/platform/

File: src/AutoSysimages.jl [hunk 3]
```diff
@@ -389,36 +392,44 @@ end
 This install the `asysimg` scripts.
 (Currently implemented only for Linux.)
 """
-function install(dir=_default_install_dir())
+function install(dir = _default_install_dir())
+
+    dir_exists = ispath(dir)
+    dir_exists || mkpath(dir)
+
     if Sys.islinux() || Sys.isapple()
```
Comment (petvana) [inline src/AutoSysimages.jl:400]: What about this?
``` julia
   if isnothing(dir)
       @warn """AutoSysimages: Installation is not yet supported for your OS.
Feel free to submit a PR."""
        return nothing
   end
```
... create dir ...
``` julia 
    script_fn = Sys.iswindows() ? "asysimg.bat" : "asysimg"
    script = joinpath(@__DIR__, "..", "scripts", "linux", script_fn)
```

Comment (petvana) [reply src/AutoSysimages.jl:400]: Ahhh, "linux".

Comment (curio-sitas) [reply src/AutoSysimages.jl:400]: there is also the "linux" directory involved in the function call

Comment (curio-sitas) [reply src/AutoSysimages.jl:400]: also, it would be more apropriate to have a `unix` directory because they both work on `linux` and `macos`

Comment (petvana) [reply src/AutoSysimages.jl:400]: I propose to rename the folder in a separate PR. Thus, I'll merge this because it looks good.

**Pull request merged**